starport/pkg/cosmosutil/peer.go

```diff
@@ -1,28 +1,28 @@
 package cosmosutil
 
 import (
+	"net"
 	"strings"
 
 	launchtypes "github.com/tendermint/spn/x/launch/types"
+	"github.com/tendermint/starport/starport/pkg/xurl"
 )
 
 // VerifyPeerFormat checks if the peer address format is valid
 func VerifyPeerFormat(peer launchtypes.Peer) bool {
 	// Check the format of the peer
 	switch conn := peer.Connection.(type) {
 	case *launchtypes.Peer_TcpAddress:
-		nodeHost := strings.Split(conn.TcpAddress, "@")
+		nodeHost := strings.Split(conn.TcpAddress, ":")
 		if len(nodeHost) != 2 ||
-			len(nodeHost[0]) == 0 ||
+			net.ParseIP(nodeHost[0]) == nil ||
```

Comment:
Not sure if validators can have domain names, if so, this change might break things.